packages/fast-components-styles-msft/src/dialog/index.ts

```diff
@@ -18,6 +18,9 @@ const styles: ComponentStyles<DialogClassNameContract, DesignSystem> = (
             '&[aria-hidden="false"]': {
                 display: "block",
             },
+            "@media (-ms-high-contrast:active)": {
+                border: "1px solid WindowText",
```

Comment:
Probably use outlineWidth to be consistent.

Reply:
updated